### ISSUE TITLE
error 2147942405 (0x80070005) when launching Nvim 0.10.3

### ISSUE THREAD

**Issue by rRobis** [neovim/neovim#31692]
### Problem

I get [error 2147942405 (0x80070005) when launching Nvim 0.10.3

It also Triggers Windows Defender workplace blocked actions (low severity). I don't have exact info on what action was blocked

The Nvim 0.10.2  version doesn't give this error

### Steps to reproduce

Download https://github.com/neovim/neovim/releases/download/v0.10.3/nvim-win64.zip

Open nvim.exe in Windows Terminal

Error  [error 2147942405 (0x80070005)] appears and nvim doesn't  launch.

### Expected behavior

Nvim should launch as expected without triggering windows defender.

### Nvim version (nvim -v)

0.10.3

### Vim (not Nvim) behaves the same?

I don't have vim

### Operating system/version

Windows 10 Business 10.0.19045 Build 19045

### Terminal name/version

Windows Terminal

### $TERM environment variable

-

### Installation

Portable Windows Build (zip)

**Comment by rRobis** [comment]
I checked, the latest nightly also has this problem for me

https://github.com/neovim/neovim/releases/tag/nightly

**Comment by justinmk** [comment]
> error 2147942405 (0x80070005)

That is the "access denied" error.

What happens if you run Nvim with `Run as administrator` ?

Separately, can you try adding an exclusion for Nvim to Windows Defender:

> Open Windows Defender Security Center, and then select Virus & threat protection > Virus & threat protection settings > Add or remove exclusions.

If you can find out how to resolve this, we will add it to our documentation.

Is there anything unusual about your Windows system, i.e. is it a corporate system with extra restrictions or "group policy"?

**Comment by rRobis** [comment]
Yes, i'm in corporate environment. I don't have admin rights.

**Comment by justinmk** [comment]
> The Nvim 0.10.2 version doesn't give this error

Oh, interesting. There are only 54 commits between 0.10.2 - 0.10.3: https://github.com/neovim/neovim/compare/v0.10.2...v0.10.3

If you can build Nvim on Windows and bisect that will help a lot. Alternatively, if our CI produced artifacts for each PR, that would really help in this kind of situation.

**Comment by rRobis** [comment]
Yes i also noticed that first one, and was wondering if it could be somehow connected

**Comment by rRobis** [comment]
For now i'm hesitant to try to build, because if i would trigger some kind of virus alert, they will again make me reinstall windows and could forbid using my special apps. If we can somehow find solution without me trying to build (i only have zig available now), then it would be great

**Comment by rRobis** [comment]
@justinmk  I just tried to run the 0.10.4 version, and i can run it without triggering the Windows Defender blocked actions.  

I installed the latest nightly, and that one is being blocked again by Windows Defender blocked actions..

**Comment by rRobis** [comment]
Been using 0.10.4 couple days now, and didn't see any unusual issues.

Strange that it got somehow fixed, and then after that got broken again in the nightly.

**Comment by justinmk** [comment]
Well, that's because these AV services "work" by labeling binaries. The nightly binary is constantly changing...
